content/learn/economics/apps.md

```diff
@@ -16,102 +16,16 @@ description: An overview of Pocket Network application economics.
-
-* In a downside scenario, where the rate for MaxRelays drops below an Application’s current locked-in rate, Applications are incentivized to keep their POKT staked to continue receiving throughput at an above market rate.
-* In an upside scenario, where the rate for MaxRelays rises above an Application’s current locked-in rate, Applications will be incentivized to unstake and restake their POKT to receive more Relays for the same amount of POKT.
-
-Regardless of the scenario, applications are able to benefit from shifts in the market for POKT making the most of their stake. 
-
-## Base Throughput
-
-BaseThroughput is the baseline number of Relays we aim for an Application to get per POKT staked, assuming no external factors influencing POKT. This is calculated as:
-
-```math
-$$
-BaseThroughput = BaseRelaysPerPOKT * StakedPOKT
-$$
-```
-
-`BaseRelaysPerPOKT` is a uint64, governed by the Pocket DAO, which describes the baseline number of Relays the Pocket DAO aims for each Application to receive per POKT staked. As a multiplier, changing this number more significantly impacts MaxRelays than changing `StabilityAdjustment`. For further granularity, `BaseRelaysPerPOKT` can be expressed as:
-
-```math
-$$
-BaseRelaysPerPOKT = BaseRelaysPerPOKTNumerator / BaseRelaysPerPOKTDenominator
-$$
-```
-
-This allows the protocol to express decimals in the form of fractional integers, enabling more granularity for the BaseRelaysPerPOKT number.
-
-Due to the oracle problem, it is not possible to automatically adjust BaseRelaysPerPOKT based on the market price of POKT. The DAO will track indicators (such as ParticipationRate as well as the rate of change of new POKT being staked on the demand side), and adjust Pocket’s economic levers, as necessary, to ensure that Relays remain affordable for Applications.
-
-To keep the real `$USDPerRelay` price as close to the `$USDPerRelay` Target as possible, the Protocol Throttling Formula multiplies `BaseThroughput` by the total `ParticipationRate` of the protocol to reflect any changes in demand for Relays, then the DAO will use the `StabilityAdjustment` in the short-term to correct deviations from the `$USDPerRelay` Target that are most likely attributed to short-term changes inherent in the random walk of the cryptocurrency/FOREX markets. If the StabilityAdjustment persists above/below zero without resetting, we can attribute the deviation from the `$USDPerRelay` Target to a more permanent change in POKT’s market value, at which point the DAO will update `BaseRelaysPerPOKT` and reset `StabilityAdjustment` to zero.
+The maximum relays an app is entitled to is related to the [`BaseRelaysPerPOKT`](/learn/protocol-parameters/#baserelaysperpokt) parameter. For example, if this parameter is `200000` then the throughput that apps are entitled to is 2,000 relays per POKT staked.
 
 The `BaseRelaysPerPOKT` will be updated at the discretion of the DAO.
```

Comment:
@eullrich Would it make senes to you if the formula from the sheet was documented here?

Reply:
Yeah defining it here would be perfect.   I think also defining baseThroughput in this section and how it relates to baseRelaysPerPokt is valuable.  Because baseThroughPut is really the contract value between that app stake and the protocol right? 

So an appstake's basethroughput value is derived using the baserelaysperpokt param. At timme of appstake, baseThroughput value is locked for the remainder of that stake, regardless if the baserelaysperpokt value changes.  Correct?

Reply:
@eullrich I'm happy to add the formula from that spreadsheet. Question is, what is the relationship between RelaysPerPOKT (cell C9) and the BaseRelaysPerPOKT parameter? They don't seem to be anywhere each other in order of magnitude, and it seems important to tie one back to the other.

Also, to eliminate any doubt, what is the relationship between BaseThroughput and BaseRelaysPerPOKT?